### PR TITLE
Update Slowy.app to 1.8

### PR DESCRIPTION
**Pull request by Cretezy**
Update Slowy.app to 1.8

### CODE REVIEW

File: Casks/slowy.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  version '1.7'
-  sha256 '0b2c6414c01d4e940e1df26034258a8401f0f302278a00ff13adeecada582c81'
+  version '1.8'
+  sha256 '3dcff6295b5612a0885b0128e47ad04ec416847c8f146637cc8e56b8d3cb511c'
 
   url "http://slowyapp.com/downloads/Slowyapp_v#{version}.dmg"
   name 'Slowy'
   homepage 'http://slowyapp.com/'
-  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
+  license :commerical
```
Comment (fanquake) [inline Casks/slowy.rb]: s/commerical/commercial

Comment (CraftThatBlock) [inline Casks/slowy.rb]: Fixed.
Sorry, it shows that I did this at like 1am :P

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by CraftThatBlock** [comment]
Will make a new one as I can't get squashing to work.

**Pull request closed without merging**